snuba/subscriptions/executor.py

```diff
@@ -28,7 +28,7 @@ def execute(
         timer = Timer("query")
         try:
             request = task.task.data.build_request(
-                self.__dataset, tick.timestamps.upper, tick.offsets.upper, timer,
+                self.__dataset, task.timestamp, tick.offsets.upper, timer,
```

Comment:
A single tick can contain multiple task instances for a single subscription if the ingress rate is slower than the task period. (We'll probably have to create some sort of synthetic activity in the topic if this happens to ensure the clock is always ticking, so to speak, to ensure things like negative thresholds can be implemented correctly by consumers in real-ish time.) In this scenario, each task would execute the exact same query (aligned to the tick upper bound) since they were triggered by the same tick instead of each query being relative to the originally scheduled execution time.

This also just makes the query behavior easier to understand to in general (at least to an outside observer), since the time bounds will be consistent with the schedule time instead of drifting based on the consumer time.